daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Engine.scala

```diff
@@ -219,7 +219,7 @@ final class Engine {
       case NodeCreate(coid @ _, coinst, optLoc @ _, sigs @ _, stks @ _, key @ _) =>
         val identifier = coinst.template
         asValueWithAbsoluteContractIds(coinst.arg).flatMap(
```

Comment:
Consider changing the type of `asValueWithAbsoluteContractIds` to also only talk about `Value` and not `VersionedValue`.

Reply:
Good catch. Thx !